auth-web/src/components/auth/create-account/AccountCreateBasic.vue

```diff
@@ -66,7 +38,7 @@
           color="primary"
           class="mr-3 save-btn"
           :loading="saving"
-          :disabled="!isFormValid() || saving  || !isBaseAddressValid"
+          :disabled="!isFormValid() || saving  || !isBaseAddressValid || !isOrgBusinessTypeValid"
```

Comment:
can't isFormValid() handle isBaseAddressValid and isOrgBusinessTypeValid, rather than adding all here ?

Reply:
yes :) good catch. refactored